workflow/snakemake_rules/CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+# Developer guide
```

Comment:
Suggestion: put this file at the top level of the repo, so we can add non-Snakemake specific sections to it later.

Reply:
Oh this was an accident... Thanks!